thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/detector/email/filter/PrecisionRecallEvaluator.java

```diff
@@ -200,6 +202,21 @@ public Properties toProperties() {
     return evals;
   }
 
+  public Map<String, Double> toDoubleMap() {
+    Map<String, Double> evals = new HashMap<>();
+    evals.put(RESPONSE_RATE, getResponseRate());
+    evals.put(PRECISION, getPrecision());
+    evals.put(WEIGHTED_PRECISION, getWeightedPrecision());
+    evals.put(RECALL, getRecall());
+    evals.put(TOTALALERTS, (double) getTotalAlerts());
+    evals.put(TOTALRESPONSES, (double) getTotalResponses());
+    evals.put(TRUEANOMALIES, (double) getTrueAnomalies());
+    evals.put(FALSEALARM, (double) getFalseAlarm());
+    evals.put(NEWTREND, (double) getTrueAnomalyNewTrend());
```

Comment:
Hi Yen, here the get methods are all return (int) rather than (double), if casting them to double, will these shown on UI as "number of anomalies = 10.0"?

Reply:
Fixed!